src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+import java.util.Random;
+
+public class Circle extends ColorSupplier {
+    private int radius;
+    private String color;
+
+    public Circle() {
+        this.radius = new Random().nextInt(4) + 1;
```

Comment:
no need to create a random value right here, you need to pass it from outside to the constructor